src/WebJobs.Script/Sanitizer.cs

```diff
@@ -32,6 +33,8 @@ internal static string Sanitize(string input)
                 return string.Empty;
             }
 
+            input = SanitizerRegex.Regex().Replace(input, SecretReplacement);
```

Comment:
Can we move this *below* the check for `MayContainCredentials()`? And maybe add a check whether ` || input.Contains(":")` to that method? That's meant to be a very quick check to see if we need to go any further and the way you have this set up now, we'd do a regex replacement always, no matter what...

Reply:
Done